builders/manage/dapp/.pages

```diff
@@ -0,0 +1,5 @@
+title: Using the Tanssi dApp
+hide: false
+nav:
+  - index.md
+  - 'Services Payment': 'services-payment.md'
```

Comment:
I don't think the "Services Payment" title is self explanatory, maybe ewe should find another title?